package.json

```diff
@@ -1,19 +1,19 @@
 {
   "name": "c32check",
-  "version": "0.0.6",
+  "version": "1.0.0",
   "description": "Crockford base-32 checksum encoding",
   "main": "lib/index",
+  "unpkg": "dist/c32check.js",
+  "jsdelivr": "dist/c32check.js",
```

Comment:
Can we add `prettier-config` ?

Reply:
`@blockstack/prettierconfig@0.0.4` has trailing comma

Reply:
Added, but did not format the files. It changes most lines so I'm thinking another PR for that